src/Randomizer.SMZ3.ChatIntegration/IChatAuthenticationService.cs

```diff
@@ -1,12 +1,13 @@
 ﻿using System.Threading;
 using System.Threading.Tasks;
+using Randomizer.SMZ3.ChatIntegration.Models;
 
 namespace Randomizer.SMZ3.ChatIntegration
 {
     public interface IChatAuthenticationService
     {
         Task<string?> GetTokenInteractivelyAsync(CancellationToken cancellationToken);
 
-        Task<string?> GetUserNameAsync(string accessToken, CancellationToken cancellationToken);
+        Task<AuthenticatedUserData?> GetUserData(string accessToken, CancellationToken cancellationToken);
```

Comment:
Oh, also, as a matter of consistency, please add Async to the name.

Reply:
Crap. I missed these comments. I'll get a new PR opened today for them. (I'll also look over your PR as apparently I forgot about it...)